app/assets/javascripts/admin/addon/components/admin-flag-item.gjs

```diff
@@ -4,13 +4,22 @@ import { fn } from "@ember/helper";
 import { on } from "@ember/modifier";
 import { action } from "@ember/object";
 import { htmlSafe } from "@ember/template";
+import DButton from "discourse/components/d-button";
 import DToggleSwitch from "discourse/components/d-toggle-switch";
+import DropdownMenu from "discourse/components/dropdown-menu";
 import { ajax } from "discourse/lib/ajax";
 import { popupAjaxError } from "discourse/lib/ajax-error";
+import i18n from "discourse-common/helpers/i18n";
+import DMenu from "float-kit/components/d-menu";
+const NOTIFY_USER_ID = 6;
```

Comment:
This would be better to put in our `discourse/app/lib/constants.js` which you change in the rake task here https://github.com/discourse/discourse/blob/62b1b69e820bfa48898d5f654c2b71162078a73a/lib/tasks/javascript.rake#L132-L132 . You could add a new `SYSTEM_FLAG_IDS` const with e.g. `notify_user`, `inappropriate`, `off_topic` and so on keys.